### ISSUE TITLE
Wrong package name in setup.py and wrong project name in PyPI

### ISSUE THREAD

**Issue by junaruga** [junaruga/rpm-py-installer#24]
https://github.com/junaruga/rpm-py-installer/blob/master/setup.py#L65

Change to `rpm-py-installer` as soon as possible.

**Comment by junaruga** [comment]
I changed `setup.py`'s name by https://github.com/junaruga/rpm-py-installer/pull/25 .

Last time the project name was `rpm_py_installer` in old PyPI [1].
I uploaded new version with new package name `rpm-py-installer` to the old PyPI.
But the project name was not changed.
So, I deleted the project page from old PyPI.
Then [1] is "page not found".
And when I ran below upload command, I got an error.
Maybe the project data is still alive internally.

```
(venv) $ python3 setup.py sdist upload
...
Submitting dist/rpm-py-installer-0.1.2.tar.gz to https://upload.pypi.org/legacy/
Upload failed (400): This filename has previously been used, you should use a different version.
error: Upload failed (400): This filename has previously been used, you should use a different version.
```
On new PyPI [2], there is still the project data with old name `rpm_py_installer`.

[1] https://pypi.python.org/pypi/rpm_py_installer
[2] https://pypi.org/project/rpm_py_installer/

**Comment by junaruga** [comment]
There is a proposal to rename the project name from Web UI.
https://github.com/pypa/warehouse/issues/1919

**Comment by junaruga** [comment]
Similar issue: https://github.com/pypa/pypi-legacy/issues/631

**Comment by junaruga** [comment]
https://www.python.org/dev/peps/pep-0008/#package-and-module-names
> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

**Comment by junaruga** [comment]
I released new version v0.1.3. And the project name is changed.
https://pypi.python.org/pypi/rpm-py-installer
https://pypi.org/project/rpm-py-installer

So, summarizing this situation, one of the available way to change project name is

1. Delete project completely from PyPI.
2. Update `setup.py` name on local.
3. Bump version and release the new version.

By the way, I did set `rpm-py-installer` as project name (`setup.py` setup.name), and `rpm_py_installer` as a directory name, referring `flake8-*` modulle or `pytest-cov`.



PS. Below is official way.
http://legacy.python.org/dev/peps/pep-0423/#how-to-rename-a-project
